Emby.Server.Implementations/Library/DefaultAuthenticationProvider.cs

```diff
@@ -165,6 +165,34 @@ public string GetPasswordHash(User user)
             return user.Password;
         }
 
+        public void ChangeEasyPassword(User user, string newPassword, string newPasswordHash)
+        {
+            ConvertPasswordFormat(user);
+
+            if (newPassword != null)
+            {
+                newPasswordHash = string.Format("$SHA1${0}", GetHashedString(user, newPassword));
+            }
+
+            if (string.IsNullOrWhiteSpace(newPasswordHash))
+            {
+                throw new ArgumentNullException(nameof(newPasswordHash));
+            }
+
+            user.EasyPassword = newPasswordHash;
+        }
+
+        public string GetEasyPasswordHash(User user)
+        {
+            // This should be removed in the future. This was added to let user login after
```

Comment:
Couldn't we write a script that fixes it? This just created technical debt no one will remember

Reply:
Do we normally write scripts for those kinda of things? Also, do we have a preference of language?

I think we could keep it for 10.3.4 and remove it in 10.4 and tell the people with broken DB to use this version to update their PIN. I could already open a PR targeting 10.4 to be sure we don't forget. What do you think?

Reply:
I do like the idea of fixing it now and in future. It’s hard to initially diagnose this issue because it can present as just a blank screen. Unless you know where the logs are, you might just think it’s a connection problem.

Reply:
> and in future

Do you mean keep this "fix" in the code forever?

Reply:
@drpandemuc, Sorry - to clarify, I agree with your thought of committing a fix to 10.4 right away

Reply:
I agree with leaving this in code and ragging it as "migration" in code and leaving it as an issue to fix after 10.4.0. Scripts are harder to get working for normap users and aren't obvious. We should just fix it in code IMO.

Reply:
We can always force someone to upgrade "through" a version to get to the latest.